Formula/parquet-cli.rb

```diff
@@ -29,10 +29,13 @@ class ParquetCli < Formula
 
   # Patches snappy to 1.1.8.3 for MacOS arm64 support, won't be needed in >= 1.13.0
   # See https://issues.apache.org/jira/browse/PARQUET-2025
-  patch do
-    url "https://github.com/apache/parquet-mr/commit/095c78fec3378189296d38fede1255b0a4d05fd4.patch?full_index=1"
-    sha256 "9a5b54275c2426a56e246bdf4b7a799d5af8efe85c2dcdc3c32e23da3101f9d7"
-  end
+  #
+  # We're using locally patch data because parquet-cli/pom.xml is linked to parquet-hadoop project
+  # remotely, which depends on a bad version of snappy-java, so we need to add a direct dependency
+  # from parquet-cli/pom.xml to snappy-java 1.1.8.3, which overrides the dependency version given
+  # from parquet-hadoop.
```

Comment:
Also, has this issue been reported upstream?

Reply:
This is not a problem upstream, the parquet-cli project will use equivalent parquet-hadoop package (version)